<launch>
  
  <node name="cmd_rotatory_startup_locker" pkg="robotino_bringup" type="cmd_rotatory_startup_locker.py" output="screen">
    <param name="cmd_topic" value="/cmd_rotatory"/>
    <param name="lock_topic" value="/twist_mux/locks/pause_rotatory"/>
    <param name="timeout" value="120.0" type="double"/>
  </node>
  
</launch>
